sdk/python/tests/test_feature_store.py

```diff
@@ -72,6 +72,24 @@ def feature_store_with_gcs_registry():
     )
 
 
+@pytest.fixture
+def feature_store_with_s3_registry():
+    import boto3
+
+    s3 = boto3.resource("s3")
+    bucket_name = f"feast-registry-test-{int(time.time() * 1000)}"
+    bucket = s3.Bucket(bucket_name)
+    s3.meta.client.head_bucket(Bucket=bucket.name)
+
+    return FeatureStore(
+        config=RepoConfig(
+            registry=f"s3://{bucket_name}/registry.db",
```

Comment:
Let's change the path to `s3://{bucket_name}/registries/{int(time.time() * 1000)}/registry.db`

Reply:
done